client/src/components/Burger/Burger.css

```diff
@@ -0,0 +1,42 @@
+.burger-line-one, .burger-line-two, .burger-line-three {
+    height: 8px;
+    width: 50px;
+    background-color: var(--black);
+    margin: 8px 0;
+    transition: .2s ease-in-out;
+    -webkit-transition: .1s ease-in-out;
+    -moz-transition: .2s ease-in-out;
+    -ms-transition: .2s ease-in-out;
+    -o-transition: .2s ease-in-out;
+}
```

Comment:
Can we give these all a shared class? Something like `.burger-line` would be good.

Reply:
I had them separate because something slightly different happens to each one. When one and three are animated, the rotating is different. Two is the only one that disappears.

Reply:
Nevermind. I see these can be all one now. I'll fix it.

Reply:
Done.